frontend/src/tests/components/Nav/AppNavbar.test.js

```diff
@@ -83,6 +83,24 @@ describe("AppNavbar tests", () => {
         await waitFor(() => expect(getByTestId("appnavbar-todos-dropdown")).toBeInTheDocument());
     });
 
+    test("renders the recommendations menu correctly", async () => {
+
+        const currentUser = currentUserFixtures.userOnly;
+        const systemInfo = systemInfoFixtures.showingBoth;
+
+        const doLogin = jest.fn();
+
+        const {getByTestId } = render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <AppNavbar currentUser={currentUser} systemInfo={systemInfo} doLogin={doLogin} />
+                </MemoryRouter>
+            </QueryClientProvider>
+        );
+
+        await waitFor(() => expect(getByTestId("appnavbar-recommendations-dropdown")).toBeInTheDocument());
```

Comment:
Also test that the list recommendations button exists. See: `renders the diningcommons menu correctly for an admin` for an example